go/saltpack/packets.go

```diff
@@ -28,7 +28,7 @@ type encryptionHeader struct {
 	Type            MessageType    `codec:"type"`
 	Ephemeral       []byte         `codec:"ephemeral"`
 	SenderSecretbox []byte         `codec:"sendersecretbox"`
-	Receivers       []receiverKeys `codec:"rcvrs"`
+	Receivers       []receiverKeys `codec:"rcvrs,omitempty"`
```

Comment:
This isn't a good idea for forwards compatibility, since we're using the `toarray` encoding format (rather than named fields).  We had an intense internal debate as to which we should use, but in the end decided that arrays were safer since we wouldn't have to deal with decodes being fooled by dictionaries that repeat keys.